src/cpp/mip/metadata/mip_meta_utils.hpp

```diff
@@ -188,4 +227,45 @@ constexpr const char* nameOfType(const TypeInfo& type)
     }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief Get a pointer to a parameter given the offset of the member in the
+///        class/struct
+///
+/// @warning This does not work for nested types, I.E. ClassType.member1.member2
+///
+/// @param offset The offset of the member within the struct/class. Use the C++
+///         offsetof or a variant to get this value
+///
+/// @param field_info Metadata field info to look up parameter info for
+///
+/// @return A pointer to the parameter info if found, otherwise nullptr
+///
+const inline ParameterInfo* getParameterInfo(const size_t offset, const FieldInfo& field_info)
```

Comment:
Make the name a little more descriptive: `getParameterInfoFromStructMemberOffset`.
Also this probably shouldn't be `inline`.